docsrc/Managing_resources.rst

```diff
@@ -292,9 +283,9 @@ it can run SQL commands or queries.
 
 	::
 
-		engine = rm.engines.get_by_name(name="engine_name")
+		engine = rm.engines.get(name="engine_name")
```

Comment:
Also remove `name=` here and below

Reply:
It's equivalent, but I don't feel strongly on the matter so I changed it everywhere.